mainnet-canisters.json

```diff
@@ -0,0 +1 @@
+{"archive":{"rev":"b43280208c32633a29657a1051660324e88a373d","sha256":"db0f094005a0e84e243f8f300236be879dcefa412c2fd36d675390caa689d88d"},"ck_btc_index":{"rev":"a3831c87440df4821b435050c8a8fcb3745d86f6","sha256":"cac207cf438df8c9fba46d4445c097f05fd8228a1eeacfe0536b7e9ddefc5f1c"},"ck_btc_ledger":{"rev":"a3831c87440df4821b435050c8a8fcb3745d86f6","sha256":"4264ce2952c4e9ff802d81a11519d5e3ffdaed4215d5831a6634e59efd72f7d8"},"ck_eth_index":{"rev":"a3831c87440df4821b435050c8a8fcb3745d86f6","sha256":"8104acad6105abb069b2dbc8289692bd63c2d110127f8e91f99db51465962606"},"ck_eth_ledger":{"rev":"a3831c87440df4821b435050c8a8fcb3745d86f6","sha256":"e5c8a297d1c0c6d2ab2253c0280aaefd6e23fe3a8a994fc64706a1f3c3116062"},"cycles-minting":{"rev":"77f48ae63af09b6538b1bf33d3accc3bc74d14f8","sha256":"3260e795bd3e446a189539ce89d44cb29f7d196b92cdd2e2c75571c062ef1e50"},"genesis-token":{"rev":"cf237434877b39d0a94fb5ef84b13ea576a225ac","sha256":"31d91cbdfa6e1aae4cc4fee4f611e25f33922bd3d336f4cdc97d511e03b264a7"},"governance":{"rev":"87343a880050ca72b1361138535211f5770dd52e","sha256":"8665830c50c9a0dddd996008e537d060a380ac7b6c22237679bd0cecc4ee1044"},"index":{"rev":"b43280208c32633a29657a1051660324e88a373d","sha256":"62bbbada301838ad0f6e371415be990ce70e36c6f11267d4ba9fac8ff09aa32d"},"ledger":{"rev":"b0ade55f7e8999e2842fe3f49df163ba224b71a2","sha256":"d0ec2cdeee48e2dbee07c59dfdc3928413de86930242fef0704ab7c1be6c7664"},"lifeline":{"rev":"a0207146be211cdff83321c99e9e70baa62733c7","sha256":"76978515223287ece643bc7ca087eb310412b737e2382a73b8ae55fcb458da5b"},"registry":{"rev":"87343a880050ca72b1361138535211f5770dd52e","sha256":"57c72469f01fd6ea8b5c5a962a1fed9b4ad550bbebdae38c29d5ad330c25c724"},"root":{"rev":"a0207146be211cdff83321c99e9e70baa62733c7","sha256":"c280a25dc565f8a42429cb5b969906c4c5a789381e98f6e11c247c91c4dfaac5"},"sns-wasm":{"rev":"87343a880050ca72b1361138535211f5770dd52e","sha256":"6dd00ebe425ba360be161c880ce3a3b3cda5a3738d6b323a9fd0366debf590ce"},"sns_archive":{"rev":"3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d","sha256":"5c595c2adc7f6d9971298fee2fa666929711e73341192ab70804c783a0eee03f"},"sns_governance":{"rev":"87343a880050ca72b1361138535211f5770dd52e","sha256":"66156ae6686ac88836c51dc0dc970800bb83c607e83d2448c90336aa0f1ed589"},"sns_index":{"rev":"3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d","sha256":"08ae5042c8e413716d04a08db886b8c6b01bb610b8197cdbe052c59538b924f0"},"sns_ledger":{"rev":"3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d","sha256":"e8942f56f9439b89b13bd8037f357126e24f1e7932cf03018243347505959fd4"},"sns_root":{"rev":"87343a880050ca72b1361138535211f5770dd52e","sha256":"d52d44b6df33de56c5d02ecc8b26fbf7452df8e426d77ecc9f2e3c98a8a70316"},"swap":{"rev":"87343a880050ca72b1361138535211f5770dd52e","sha256":"4ea01d425cd9c6c0a2c4988af03710d7c2377527eb0375067c69a9baff9963f2"}}
```

Comment:
Any chance to have this pretty-printed such that future diffs will be shorter?

Reply:
good call about the diff! we could simply replace `to_writer` to [`to_writer_pretty`](https://docs.rs/serde_json/latest/serde_json/fn.to_writer_pretty.html). Would that work for you?

Reply:
Yes just suggested the same below